### ISSUE TITLE
X chromosome producing missing values

### ISSUE THREAD

**Issue by ealopera** [weizhouUMICH/SAIGE#233]
Hello,
I am trying to run SAIGE 0.39 in in some binary phenotypes with about 27K samples. my file for X chromosome has dosage field changed so that man are coded as 0,2 and women 0,1,2. I have used this file for other binary traits, it works in some, but in some others it gives this error message:

```
Error in if (abs(as.numeric(unlist(out.score["Tstat"])[1])/sqrt(as.numeric(unlist(out.score["var1"])[1]))) <  : 
  missing value where TRUE/FALSE needed
Calls: SPAGMMATtest -> scoreTest_SAIGE_binaryTrait_cond_sparseSigma
In addition: Warning message:
In sqrt(as.numeric(unlist(out.score["var1"])[1])) : NaNs produced
Execution halted
```

do you have any idea where this comes from?
thank you

**Comment by weizhouUMICH** [comment]
Hi @ealopera, 

This error has been fixed in the 0.39.2 
https://github.com/weizhouUMICH/SAIGE/tree/0.39.2
This version will be merged to the master branch soon. 

Thanks,
Wei

**Comment by weizhouUMICH** [comment]
We have just released a new version 1.0.0. It has computational efficiency improvements for both Step 1 and Step 2 for single-variant and set-based tests. We have created a new program github page https://github.com/saigegit/SAIGE with the documentation provided https://saigegit.github.io/SAIGE-doc/
The program will be maintained by multiple SAIGE developers there. The docker image has been updated. Please feel free to try the version 1.0.0 and report issues if any.

Thanks!
Wei
